### PR TITLE
Remove paragraphs in README.md

### PR DESCRIPTION
**Pull request by erikn69**
Same as https://github.com/spatie/package-skeleton-laravel/pull/167
https://github.com/spatie/package-skeleton-laravel/blob/e37a2bd3f222332f81c5e7d3b76c250a7752e8a6/configure.php#L103-L109

Automatic remove paragraphs inside of README.md when they are inside of `<!--delete--><!--/delete-->`

### CODE REVIEW

File: configure.php [hunk 1]
```diff
@@ -60,6 +60,27 @@ function replace_in_file(string $file, array $replacements): void {
     );
 }
 
+function remove_readme_paragraphs(string $file): void {
```
Comment (freekmurze) [inline configure.php:63]: Use camelCase, just like the other functions.

Comment (erikn69) [reply configure.php:63]: I did copy from here, but i will change it
https://github.com/spatie/package-skeleton-php/blob/ea4bc716df055b36cb8b19236fc042739d710b3b/configure.php#L50-L55

Comment (erikn69) [reply configure.php]: Fixed

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**